packages/actionmenu/src/react/__specs__/index.spec.js

```diff
@@ -57,5 +57,3 @@ describe('ActionMenu', () => {
     fireEvent.keyDown(opt, { key: 'Enter', code: 13 })
   })
 })
-
-afterEach(cleanup)
```

Comment:
Why is this no longer required?

Reply:
breaking change in testing-library. it's no longer needed or available